pandas/tests/groupby/test_groupby.py

```diff
@@ -433,6 +433,33 @@ def test_frame_groupby_columns(tsframe):
         assert len(v.columns) == 2
 
 
+@pytest.mark.parametrize(
+    "func, args",
+    [
+        ("sum", []),
```

Comment:
DO we need all these parametrezations? Or does just a few work? `sum` and `nth`?

Reply:
Probably not.  It looks like one of [sum, prod, min, max, first, last] and then nth would suffice.

Reply:
We have a `reduction_func` fixture I think you should use here; would actually be more functions but also more comprehensive and consistent with some other tests